src/components/Markdown.svelte

```diff
@@ -40,11 +43,15 @@
   function navigateToMarkdownLink(event: any) {
     if (event.target.matches(".file-link")) {
       event.preventDefault();
-      if ($activeRouteStore.resource === "projects") {
-        void updateProjectRoute({
+      void router.push({
+        resource: "projects",
```

Comment:
I believe we need to add the `peer` and `revision` properties here. The following test fails

1. Go to the awesome-radicle project.
2. Select the peer “dave” and one of the branches starting with “patches/”
3. Click on `macos-launch-agents`
4. Expected to see the the file `launchers/macos/​README.md` for the peer “dave” and the selected branch